### PR TITLE
[vcpkg] Add nologo to windows toolchain 

### LINKED ISSUE #11144: [vcpkg] Add /nologo to build flags on windows
Why?
- Less logo more meat in the logs.
- smaller log files
- compiler errors easier to spot

### PR DESCRIPTION
**Pull request by Neumann-A**
closes #11144

There are still some linker logos in the log but i am apparently unable to find the correct cmake variable for it

### CODE REVIEW

**Comment by cenit** [comment]
just for symmetry reasons, do you maybe also need to add it to `CMAKE_SHARED_LINKER_FLAGS_DEBUG` and `CMAKE_EXE_LINKER_FLAGS_DEBUG`?

Also, why is there a `/DEBUG` flag in a `*_RELEASE` symbol? I realize it was already there, just questioning it in principle

File: ports/fftw3/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: fftw3
-Version: 3.3.8-6
+Version: 3.3.8-7
```
Comment (LilyWangL) [inline ports/fftw3/CONTROL:2]: Thanks for your PR. `fftw3` has nothing change, so the version doesn't need modify.

**Review by LilyWangL — DISMISSED** [review]

**Comment by strega-nil** [comment]
@Neumann-A why do you have two different styles of adding this flag? both `set` and `string(APPEND`? Is there a reason for it?

**Comment by Neumann-A** [comment]
> @Neumann-A why do you have two different styles of adding this flag? both set and string(APPEND? Is there a reason for it?

`_INIT` flags versus normal flags. The `set` command completely overwrites the other variables and `_INIT` flags are used to initialize the normal values. I am just following the style of the other toolchains which use _INIT with string(APPEND)

https://github.com/microsoft/vcpkg/blob/305785ba9b2b4ebd37d0a2c9676b863bb7e0a94e/scripts/toolchains/linux.cmake#L12-L22

**Comment by strega-nil** [comment]
Ah, I see! why not modify the `CMAKE_EXE_LINKER_FLAGS_RELEASE_INIT` and `CMAKE_SHARED_LINKER_FLAGS_RELEASE_INIT` then?

**Comment by Neumann-A** [comment]
Because it would require me to check the CMake source again to check how to change
```
    set(CMAKE_SHARED_LINKER_FLAGS_RELEASE "/nologo /DEBUG /INCREMENTAL:NO /OPT:REF /OPT:ICF ${VCPKG_LINKER_FLAGS}" CACHE STRING "")
    set(CMAKE_EXE_LINKER_FLAGS_RELEASE "/nologo /DEBUG /INCREMENTAL:NO /OPT:REF /OPT:ICF ${VCPKG_LINKER_FLAGS}" CACHE STRING "")
```
into the `CMAKE_EXE_LINKER_FLAGS_RELEASE_INIT|CMAKE_SHARED_LINKER_FLAGS_RELEASE_INIT ` variables if this is even possible. Since the last time I changed those variables I broke vcpkg (da0a15485256837569d0e6b00a5940757ff57971 9b44e4768bf25e1ae07e6eaba072c1a1a160f978) I don't dare to touch them again for some time ;)

**Comment by cenit** [comment]
I'd trigger a full rebuild before merging, just in case ;)

**Comment by cenit** [comment]
@strega-nil (something that a toolchain modification should already do 😉)

**Review by dan-shaw — APPROVED** [review]

**Pull request merged**